include/node.h

```diff
@@ -42,73 +42,63 @@ class Node {
 
   //! Assign hydraulic head at the node
   //! \param[in] head hydraulic head at the node
-  void head(double head) {
+  void elevation(double head) {
+    elevation_ = head;
     head_ = head;
-    iter_head_ = head;
     ishead_ = true;
   }
-  void iter_head(double iter_head) { iter_head_ = iter_head; }
-
-  bool isres() const { return isres_; }
+  void head(double iter_head) { head_ = iter_head; }
 
   //! Return hydraulic head
   //! \retval head_ hydraulic head at the node
+  double elevation() const { return elevation_; }
   double head() const { return head_; }
-  double iter_head() const { return iter_head_; }
 
   //! Return head assignment status
   //! \retval ishead_ head assignment status at the node
   bool ishead() const { return ishead_; }
```

Comment:
Does this mean we can have both a head and a reservoir?

Reply:
I dont think we need ishead() anymore. I will remove it

Reply:
OK